harvester/core/models/datatypes/document.py

```diff
@@ -8,18 +8,34 @@
 PRIVATE_PROPERTIES = ["pipeline", "from_youtube", "lowest_educational_level"]
 
 
+class DocumentManager(models.Manager):
+
+    def build_from_seed(self, seed, collection=None, metadata_pipeline_key=None):
+        properties = copy(seed)  # TODO: use setters that update the pipeline?
```

Comment:
moet dit nu gebeuren of kan dat later?

Reply:
Dit kan later. Voorlopig gaat Tika alleen 's nachts werken. Als we het tussendoor willen, dan wordt dit belangrijk.